packages/core/protocols/src/proto/dxos/halo/credentials.proto

```diff
@@ -181,6 +181,11 @@ message DeviceProfile {
   DeviceProfileDocument profile = 1;
 }
 
+/// [ASSERTION]: Sets default space for identity.
+message DefaultSpace {
+  dxos.keys.PublicKey space_key = 1;
```

Comment:
string spaceId

Reply:
lets prefer space ids everywhere outside of internal halo state machiene when referring to a space